core/config.go

```diff
@@ -45,7 +52,7 @@ func NewConfig(opts ...ConfigOption) *Config {
 	d := &Config{
 		configFolder: DefaultConfigFolder(),
 		dkgTimeout:   DefaultDKGTimeout,
-		//certmanager: net.NewCertManager(),
+		// certmanager: net.NewCertManager(),
```

Comment:
We should probably remove this completely?

Reply:
We want to deprecate TLS termination inside drand anyway, so kill it with 🔥 !